test/integration/suites/node-attestation/03-test-node-attestation

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Test node attestation api
+jointoken=`docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep jointoken`
+echo "Created Join Token" $jointoken
+
+svid1=`docker-compose exec -u 1001 -T spire-agent /opt/spire/conf/agent/node-attestation -testStep jointokenattest -tokenName $jointoken`
+if [[ $? -ne 0 ]];
+then
+	fail-now "Failed to do initial join token attestation"
+fi
+echo "Received initial SVID:" $svid1
+
+svid2=`docker-compose exec -u 1001 -T spire-agent /opt/spire/conf/agent/node-attestation -testStep renew -certificate "${svid1}"`
+if [[ $? -ne 0 ]];
+then
+	fail-now "Failed to do SVID renewal"
+fi
+echo "Received renewed SVID:" $svid2
+
+docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep ban -tokenName ${jointoken} 
+if [[ $? -ne 0 ]];
+then
+	fail-now "Failed to do initial join token attestation"
+fi
+echo "Agent banned"
+
+if docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep renew -certificate "${svid2}" 
+then
+	fail-now "OOOPS"
+fi
+
+#docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep renew -certificate "${svid2}"
```

Comment:
I think we can add more steps if working we do attestion with x509 pop, for example we can ban and verify we has no longer access then, we can remove banned entry that will allow to attest it again, that is something we wanted when adding ban basically a way to prevent reattestation, but that is allowed once we delete banned entry, do you mind to add and example like that?

Reply:
possible steps::
Attest agent
delete agent
reattest 
ban agent 
reatest (fails)
delete agent 
reatest (success)